buku.py

```diff
@@ -3259,6 +3294,22 @@ def main():
         # Enable color in logs
         setup_logger(logger)
 
+    # Set colors
+    if args.colorstr:
+        ID_str = '%d. %s [%s]\n'
+        ID_DB_str = '%d. %s'
+        MUTE_str = '%s (L)\n'
+        URL_str = '%s   > %s\n'
+        DESC_str = '%s   + %s\n'
+        TAG_str = '%s   # %s\n'
+        colors = Colors(*[COLORMAP[c] for c in args.colorstr], reset=COLORMAP['x'])
+        ID_DB_str = colors.ID_DB_str + ID_DB_str + COLORMAP['x']
+        URL_str = colors.URL_str + URL_str + COLORMAP['x']
+        DESC_str = colors.DESC_str + DESC_str + COLORMAP['x']
+        TAG_str = colors.TAG_str + TAG_str + COLORMAP['x']
+    else:
+        print('no')
```

Comment:
Perhaps this should be a more explicit message? I'm not sure of the case that would trigger this, though. It seems like it might not be needed, since `args.colorstr` has default value.